src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/DependencyAnalysis/SealedVTableNode.cs

```diff
@@ -201,7 +201,7 @@ protected override DependencyList ComputeNonRelocationBasedDependencies(NodeFact
         public override ObjectData GetData(NodeFactory factory, bool relocsOnly)
         {
             ObjectDataBuilder objData = new ObjectDataBuilder(factory, relocsOnly);
-            objData.RequireInitialAlignment(4);
+            builder.RequireInitialAlignment(factory.Target.SupportsRelativePointers ? 4 : factory.Target.PointerSize);
```

Comment:
```suggestion
            objData.RequireInitialAlignment(factory.Target.SupportsRelativePointers ? 4 : factory.Target.PointerSize);
```
then ilc publishes with `SupportsRelativePointers: false` and `Configuration: Release`. 🙂

Reply:
Oops. Thanks!